src/lib/Components/Consignments/Screens/__tests__/SelectFromPhotoLibrary-tests.tsx

```diff
@@ -8,6 +8,9 @@ jest.mock("lib/NativeModules/triggerCamera", () => ({ triggerCamera: jest.fn() }
 import { triggerCamera } from "lib/NativeModules/triggerCamera"
 const triggerMock = triggerCamera as jest.Mock<any>
 
+jest.mock("Alert", () => ({ alert: jest.fn() }))
+jest.mock("Linking", () => ({ openURL: jest.fn() }))
```

Comment:
Interesting, never seen this technique

Reply:
When I first saw it mentioned, I assumed people just mis-wrote what they actually meant, but nope this definitely is how it’s done. I guess it’s referring to pre-defined mocks?

Reply:
Yeah, I was thinking that - because React Native has this kind of setup too: https://github.com/facebook/react-native/blob/master/jest/setup.js#L36-L44

Reply:
Maybe this is related to Haste?